app/models/nearby_facility.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'facilities/client'
+
+class NearbyFacility < BaseFacility
+  class << self
+    attr_writer :validate_on_load
+
+    def query(params)
+      require 'json'
```

Comment:
surprises me that you have to require json.. could be that the other facility stuff uses `ox` as the parser.. not a big deal.. just surprised it wasn't included .. are you doing so in the method because it will go away after the real implementation? if it's staying should you move it to the top of the file?

Reply:
This is only for returning the mocked request and will be removed once the actual code is in place. I just checked and you are correct that `require 'json'` is unnecessary.